pkg/sdn/plugin/ovscontroller_test.go

```diff
@@ -212,3 +212,402 @@ func TestOVSService(t *testing.T) {
 		t.Fatalf("Unexpected flow changes: %v\nOrig: %v\nNew: %v", err, origFlows, flows)
 	}
 }
+
+var enp1 = osapi.EgressNetworkPolicy{
+	TypeMeta: kapiunversioned.TypeMeta{
+		Kind: "EgressNetworkPolicy",
+	},
+	ObjectMeta: kapi.ObjectMeta{
+		Name: "enp1",
+	},
+	Spec: osapi.EgressNetworkPolicySpec{
+		Egress: []osapi.EgressNetworkPolicyRule{
+			{
+				Type: osapi.EgressNetworkPolicyRuleAllow,
+				To: osapi.EgressNetworkPolicyPeer{
+					CIDRSelector: "192.168.0.0/16",
+				},
+			},
+			{
+				Type: osapi.EgressNetworkPolicyRuleDeny,
+				To: osapi.EgressNetworkPolicyPeer{
+					CIDRSelector: "192.168.1.0/24",
+				},
+			},
+			{
+				Type: osapi.EgressNetworkPolicyRuleAllow,
+				To: osapi.EgressNetworkPolicyPeer{
+					CIDRSelector: "192.168.1.1/32",
```

Comment:
I will add more tests with 'dnsName' field in EgressNetworkPolicyPeer, once this pr and egress dns support pr are merged.